modules/order/src/Plugin/Field/FieldType/AdjustmentItem.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+namespace Drupal\commerce_order\Plugin\Field\FieldType;
+
+use Drupal\commerce_order\Adjustment;
+use Drupal\Core\Field\FieldItemBase;
+use Drupal\Core\Field\FieldStorageDefinitionInterface;
+use Drupal\Core\TypedData\DataDefinition;
+
+/**
+ * Plugin implementation of the 'commerce_adjustment' field type.
+ *
+ * @FieldType(
+ *   id = "commerce_adjustment",
+ *   label = @Translation("Adjustment"),
+ *   description = @Translation("Stores an adjustment which contains a decimal amount and a three letter currency code and other metadata."),
+ *   list_class = "\Drupal\commerce_order\Plugin\Field\FieldType\AdjustmentItemList",
+ *   default_widget = "commerce_adjustment_default_widget",
+ * )
+ */
+class AdjustmentItem extends FieldItemBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  public static function propertyDefinitions(FieldStorageDefinitionInterface $field_definition) {
+    $properties['value'] = DataDefinition::create('any')
+      ->setLabel(t('Adjustment value object.'))
```

Comment:
"Value" is probably good enough as a label? The column is definitely not storing an object, if we're to be precise.